docker-compose.yaml

```diff
@@ -0,0 +1,26 @@
+version: "3.8"
+
+secrets:
+  config_json:
+    # Create this file from secrets.dist.json
+    file: secrets.json
+
+services:
+  foundry-dev:
```

Comment:
foundry-tests? foundry-cypress? this won't be my dev instance, it'll be exclusively for our tests

Reply:
I just need it to not be foundry, since I use that. Are you okay with foundry-test?

Reply:
I'd prefer plural to better communicate that it's for running _tests_ and not _a test_ itself, but yeah. Don't hate it.

Reply:
I was  thinking test like test environment. If you don't hate it, I'll use it since I have a feeling I'm more likely to interact with this environment in other contexts.

Reply:
I guess I'll file another PR?

Reply:
oh oops lol

Reply:
sorry